.changeset/hungry-crabs-dress.md

```diff
@@ -0,0 +1,5 @@
+---
+"saleor-dashboard": patch
```

Comment:
suggestion: IMO this should be minor as this change is not needed to be released in 3.19